superset-frontend/src/chart/ChartRenderer.jsx

```diff
@@ -95,7 +95,6 @@ class ChartRenderer extends React.Component {
         this.hasQueryResponseChange ||
         !isEqual(nextProps.datasource, this.props.datasource) ||
         nextProps.annotationData !== this.props.annotationData ||
-        nextProps.ownState !== this.props.ownState ||
```

Comment:
We need to work on refactoring this chain of "OR" statements in a single return. @EvilDrW, can you give me a bit of context as to why we have so many conditionals there in the first place?